jetpack-beta.php

```diff
@@ -1078,6 +1080,23 @@ static function run_autoupdate() {
 
 		}
 	}
+
+	/**
+	 * This checks intends to fix errors in our build server when jetpack
+	 * @param $source
+	 * @param $remote_source
+	 *
+	 * @return WP_Error
+	 */
+	static function check_for_main_file( $source, $remote_source ) {
+		if ( $source === $remote_source . '/jetpack-dev/' ) {
```

Comment:
can we also check for the existence of `_inc/build/static.html`? That will confirm that the front end built too.

Reply:
Addressed in 56883a7f0743ed25838f63ea7981ae2e40b61dc8